docs/user-docs/latest/plugins/strapi-plugins.md

```diff
@@ -51,5 +56,85 @@ The Users & Permissions plugin impacts several parts of the admin panel. The tab
\ No newline at end of file
+| Content-type Builder | <ul>Creation of a default collection type "User" which allows for the management of the end users, the end-user roles and their permissions. This collection type cannot be deleted and the composing fields cannot be edited, but the addition of new fields is possible. </ul> |
+| Content Manager | <ul>Addition of the default "User" collection type that allows for the managment of end-user accounts (see [Managing end-user accounts](../users-roles-permissions/managing-end-users.md)). <ul><li>By default, the following fields are available: Username, Email, Password, as well as Confirmed and Blocked as boolean fields.</li> <li>The "User" collection type has a relation established with the "Role" collection type. All end-user accounts must have a designated role: by default, the end user is attributed the end-user role set as default, but that role can be changed via the end-user entries directly in the Content Manager.</li></ul> </ul> |
+
+### <img width="28" src="../assets/plugins/EmailPlugin.png"> Email plugin
+
+The Email plugin allows users to send email from the server or from external providers such as Sendgrid. The Email plugin is not configurable in the admin panel, however users can test email delivery if it has been setup by an administrator. More information about the email plugin is available in the [Developer Documentation](/developer-docs/latest/plugins/email.md).
+
+| Section impacted | Options and settings |
+|------------------|----------------------|
+| Settings         | <ul><li>Addition of "Email plugin" setting section, which contains a "Configuration" sub-section. In the Configuration section, only the email address field under "Test email delivery" is modifiable by users. A **send test email** button sends a test email.</li> <li>Addition of "Email" to the permissions for authenticated and public users. In the Email section the ability to send emails via the API can be enabled or disabled. <br>👉 Path reminder: ![Settings icon](../assets/icons/settings.svg) *Settings > Users and Permissions > Roles* </li></ul>|
+
+## Additional plugins
+
+### <img width="28" src="../assets/plugins/Documentation-swagger.png"> Documentation
+
+ The Documentation plugin automates documentation for APIs in a Strapi application using the Open API specification version 3.0.1. When the Documentation plugin is installed it is available in the admin panel, under the heading plugins. The Documentation plugin is available in the in-app Marketplace and the [Strapi Market](https://market.strapi.io/plugins/@strapi-plugin-documentation).  The Documentation plugin allows you to:
+
+- open the API documentation,
+- regenerate the documentation,
+- restrict access to the documentation endpoint.
+
+The Documentation plugin affects multiple parts of the admin panel. The table below lists all the additional options and settings that are added to a Strapi application once the plugin has been installed.
```

Comment:
I just noticed that this introductory sentence before the table is not used for all plugin docs, maybe it would be better to add it everywhere there's a table (as always, for more clarity & consistency) 🤓

Reply:
done